### ISSUE TITLE
Guake start script uses wrong tty

### ISSUE THREAD

**Issue by electrodude** [Guake/guake#652]
Any program specified in "Path to script executed on Guake start" gets run on Guake's controlling tty, not on the tty in the first tab Guake opens.

In other words, if you put `echo 'Hello World!'` there, quit guake, and run `guake` in a non-Guake terminal, the text `Hello World!` will appear on the non-Guake terminal that you typed `guake` into, but when you look at Guake, all you'll see is an empty prompt.

**Comment by Stibbons** [comment]
indeed, this is not executed inside guake, it's just a way to execute a script at startup. Will be reworked in 1.0
